src/app/components/modules/Header.jsx

```diff
@@ -39,7 +37,7 @@ class Header extends React.Component {
         if (nextProps.location.pathname !== this.props.location.pathname) {
             const route = resolveRoute(nextProps.location.pathname);
             if (route && route.page === 'PostsIndex' && route.params && route.params.length > 0) {
-                const sort_order = route.params[0] !== 'home' ? route.params[0] : null;
+                const sort_order = route.params[1] !== 'home' ? route.params[1] : null;
                 if (sort_order) window.last_sort_order = this.last_sort_order = sort_order;
```

Comment:
no need to multi-assign

also... why are we using window here? why not use redux state? window.last_sort_order is used in TagList.jsx

Reply:
Hmm.. don't remember exactly why I used window here, probably it was even before I've started using redux.